app/dashboard/tip_views.py

```diff
@@ -234,7 +220,7 @@ def receive_tip_v2(request, pk, txid, network):
 @csrf_exempt
 @ratelimit(key='ip', rate='5/m', method=ratelimit.UNSAFE, block=True)
 def send_tip_4(request):
-    """Handle the fourth stage of sending a tip (the POST)
+    """Handle the fourth stage of sending a tip (the POST).
```

Comment:
🥇

Reply:
I mean... It's pep-257...  Sorry, I don't like having 100s of linter warnings showing up all over the place.

Reply:
@mbeacom this makes perfect sense and I really encourage and like fixing of linting issues. We are on the same page really. #respect 🙇 And I am serious, no sarcasm implied. Sorry if it sounded otherwise.